docs/content/en/docs/concepts/support-versions.md

```diff
@@ -112,12 +112,15 @@ However, if you create your management cluster with version v0.15.0 and then upg
 
 No. We perform regular upgrade reliability testing for sequential version upgrade (e.g. going from version 0.9 to 0.10, then from version 0.10 to 0.11), but we do not perform testing on non-sequential upgrade path (e.g. going from version 0.9 directly to 0.11). You should _not_ skip minor versions during cluster upgrade. However, you can choose to skip patch versions.
 
-### What kind of fixes are back-ported to the previous versions?*
+### What kind of fixes are back-ported to the previous versions?
 Back-ported fixes include CVE patches and bug fixes for the Amazon EKS Anywhere components and the Kubernetes versions that are supported by the corresponding Amazon EKS Anywhere versions. 
```

Comment:
Back-ported fixes include CVE patches and bug fixes for EKS Anywhere components and the Kubernetes versions that are supported by the corresponding EKS Anywhere version.